src/caret_analyze/record/record_factory.py

```diff
@@ -29,7 +29,7 @@
     print('Succeed to find record_cpp_impl. the C++ version will be used.')
 except ModuleNotFoundError:
     use_cpp_impl = False
```

Comment:
If removing `use_cpp_impl`,  some method e.g. `is_cpp_impl_valid` is not available.

Reply:
I think `is_cpp_impl_valid` is no longer needed since we removed `record.py`.

Reply:
Because not using `is_cpp_impl_valid` in caret, I delete this functions.
Fixed in https://github.com/tier4/caret_analyze/pull/493/commits/528e7a9bf664cf6930816068a69d1f823a7336cb.